modular_skyrat/modules/customization/modules/mob/living/carbon/human/species/humanoid.dm

```diff
@@ -35,3 +35,11 @@
 
 /datum/species/humanoid/get_species_lore()
 	return list("Make sure you fill out your own custom species lore!")
+
+/datum/species/humanoid/prepare_human_for_preview(mob/living/carbon/human/human)
+	var/main_color = skintone2hex("caucasian1")
+	human.dna.features["mcolor"] = main_color
+	human.dna.features["mcolor2"] = main_color
+	human.dna.features["mcolor3"] = main_color
+	human.update_mutant_bodyparts(TRUE)
+	human.update_body(TRUE)
```

Comment:
Could probably go for some hair too.

That being said, isn't humanoid intended to be used for species that _don't_ have skintones that match humans?

Reply:
Yea, I was gonna say the same, could we use like, a greenish or reddish or something, make it look like an Orc or Devil of some sort? Would help emphasize it as humanOID and not just human